packages/amplify-provider-awscloudformation/src/iterative-deployment/deployment-manager.ts

```diff
@@ -199,7 +272,8 @@ export class DeploymentManager {
     try {
       await this.deploymentStateManager?.startRollback();
       await this.deploymentStateManager?.startCurrentStep();
-    } catch {
+    } catch (err) {
+      console.log(err);
```

Comment:
context.print(), or maybe context.warning()

Reply:
This shouldn't be here since we don't stop if we can't update the state manager